Project.toml

```diff
@@ -7,32 +7,33 @@ version = "0.1.8"
 ACSets = "227ef7b5-1206-438b-ac65-934d6da304b8"
```

Comment:
```suggestion
```

While we have this PR open, do we need ACSets as a dependency of AlgebraicDynamics? I thought all of these exports were reexported by Catlab already

Reply:
ok yeah I didn't realize that if you have Catlab, you don't need ACSets.